packages/lesswrong/components/users/DraftsPage.tsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
```

Comment:
An entirely new page (and, in a separate component + change, context menu item linking to this page) for a user's drafts